docs/understanding-your-pipeline/canonical-event/index.md

```diff
@@ -42,280 +48,403 @@ The sections below go over the standard fields found in all Snowplow events.
+| `ti_category`       | text     | Product category                                           | No        | 'New Age'       |
+| `ti_price`          | decimal  | Product unit price                                         | Yes       | 9.99            |
+| `ti_price_base`*    | decimal  | Price in base currency                                     | No        | 9.99            |
+| `ti_quantity`       | integer  | Number of product in transaction                           | Yes       | 2               |
+| `ti_currency`       | text     | Currency                                                   | No        | 'EUR'           |
+| `base_currency`*    | text     | Reporting currency                                         | No        | 'GBP'           |
 
 \* Set exclusively by the [Currency conversion enrichment](/docs/enriching-your-data/available-enrichments/currency-conversion-enrichment/index.md).
 
 #### Custom structured events
 
-:::note
+If you wish to track an event that Snowplow does not recognize as a first class citizen (i.e. one of the events listed above), then you can track them using the generic 'custom structured events'. There are five fields that are available to store data related to custom events:
 
-Currently, custom events are not available in [BDP Cloud](/docs/getting-started-with-snowplow-bdp/cloud/index.md).
+| **Field**     | **Type** | **Description**                                                                                | **Reqd?** | **Example**                   |
+| ------------- | -------- | ---------------------------------------------------------------------------------------------- | --------- | ----------------------------- |
+| `se_category` | text     | Category of event                                                                              | Yes*       | 'ecomm', 'video'              |
+| `se_action`   | text     | Action performed / event name                                                                  | Yes*       | 'add-to-basket', 'play-video' |
+| `se_label`    | text     | The object of the action e.g. the ID of the video played or SKU of the product added-to-basket | No        | 'pbz00123'                    |
+| `se_property` | text     | A property associated with the object of the action                                            | No        | 'HD', 'large'                 |
+| `se_value`    | decimal  | A value associated with the event / action e.g. the value of goods added-to-basket             | No        | 9.99                          |
 
-:::
 
-If you wish to track an event that Snowplow does not recognise as a first class citizen (i.e. one of the events listed above), then you can track them using the generic 'custom structured events'. There are five fields that are available to store data related to custom events:
+\* These fields are only required for `struct` events.
+
+#### Custom unstructured (self-describing) events
```

Comment:
Should we flip this to "Custom self-describing (unstructured) events"? (though this might break some links... :/)

Reply:
Links be damned